c2corg_ui/format/ltag.py

```diff
@@ -16,18 +16,27 @@
 class LTagPreprocessor(Preprocessor):
     """ Preprocess lines to clean LTag blocks """
 
+    """ Supported tags are L#, R# """
+    RE_LTAG_UNSUPPORTED = re.compile(r'[L|R]#[^\s|]+')
```

Comment:
Nop this regex is checking in the whole block if an **un**supported tag is present :)